app/main.py

```diff
@@ -1,2 +1,51 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = [[] for _ in range(self.capacity)]
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if self.length == self.threshold:
+            self.resize()
+        hash_ = hash(key)
+        index = hash_ % self.capacity
+        while True:
+            if not self.hash_table[index]:
+                self.hash_table[index] = [key, value, hash_]
+                self.length += 1
+                break
+
+            if self.hash_table[index][0] == key:
+                self.hash_table[index][1] = value
+                break
+
+            index = (index + 1) % self.capacity
+
+    def __getitem__(self, key: int) -> Any:
+        hash_ = hash(key)
+        index = hash_ % self.capacity
+        while self.hash_table[index]:
+            if (
+                self.hash_table[index][2] == hash_
+                and self.hash_table[index][0] == key
+            ):
+                return self.hash_table[index][1]
+            index = (index + 1) % self.capacity
+        raise KeyError(key)
+
+    def __len__(self) -> int:
+        return self.length
+
```

Comment:
I recomend implement some of this methods 
![image](https://user-images.githubusercontent.com/58142395/218783857-264463ea-c55e-4f0f-934e-53aa3bad7c68.png)

Reply:
thought not necessarily